src/components/App.js

```diff
@@ -9,6 +9,7 @@ import Community from './Community';
 import CodeOfConduct from './CodeOfConduct';
 import Contact from './Contact';
 import '../styles/App.css';
+import '../styles/diversitynow.css';
```

Comment:
does this raise an error for you locally? this is what I get:
```
Module not found: Can't resolve '../styles/diversitynow.css' in '/Users/mdecuir/projects/mplsjrdevs/src/components'
```

Reply:
The url needs the ../ in front of it for the picture location, to resolve that error.

Reply:
If you change the import to be pointing to `.scss` and rename `diversitynow.css` to `diversitynow.scss`, does this still work? If so, do that, because that's compatible with `node-sass` coming with my PR

Reply:
Got it - looks good. I'll have to change the `.css` to `.scss` in my PR, but that requires `node-sass` so it won't work if you try to change it here.